static/templates/stream_does_not_exist_error.hbs

```diff
@@ -0,0 +1,9 @@
+{{#> compose_banner }}
```

Comment:
Is it easy to put these special error message templates in a `static/templates/compose_banner/` directory? We've been trying to clear the root directory of this sort of error message template, and I think that would save us from wanting to name this `compose_banner_stream_does_not...` for clarity.

Reply:
that sounds great! I was also not very happy with the `compose_` prefix here